recipes/finetuning-scheduler/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "finetuning-scheduler" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/finetuning-scheduler-{{ version }}.tar.gz
+  sha256: f073fb7cb89e7ba4efe9f79079ea36859424589fcb1a4e61c3b70f96d58996f1
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
This can probably be `noarch` instead. Can you check? Even if it fails Windows due to the lack of pytorch the noarch recipe is preferred.

Reply:
Awesome, thanks so much @ocefpaf! I suspected that might be the case and had been planning to switch back to noarch in feedstock but I was trying to get all the checks to pass for this initial PR :facepalm: . 

I've gone ahead and switched to noaarch and the two expected builds are passing (macOS and Linux) with win_64 failing as expected. Let me know if there's anything else you need. Thanks again! :partying_face: